index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="description" content="Hello Studio Website.">
+    <title>Hello Studio</title>
+  </head>
+  <body>
+    <header>
+      <h1>
+        <span>the time is</span>
+        <span>now</span>
+      </h1>
+      <nav>
+        <ul>
+          <li><a href="/">intro</a></li>
+          <li><a href="/">gallery</a></li>
+          <li><a href="/">services</a></li>
+          <li><a href="/">contact</a></li>
+        </ul>
+      </nav>
+      <section>
+        <ul>
+          <li><a href="https://pinterest.com"><img src="assets/pinterest.svg" aria-hidden="true"></a></li>
```

Comment:
It should also have text inside it to be understandable from the perspective of a person using screenreader. So we should have `<a href="https://pinterest.com">Pinterest<img src="assets/pinterest.svg" aria-hidden="true"></a>` and then we will hide text with CSS.

Reply:
👍